bundles/CoreBundle/Resources/config/commands.yml

```diff
@@ -0,0 +1,6 @@
+services:
+    Pimcore\Bundle\CoreBundle\Command\ThumbnailsOptimizeImagesCommand:
```

Comment:
That shouldn't be necessary because it's autoregistered and autowired, right? Or any other reason for that?

Reply:
well, it would be autoconfigured/autowired if the path is configured, but Pimcore doesn't do that. Symfony does automatically recognize commands, but thats deprecated and they should be registered using the container starting with 4.0

Reply:
Ok thanks, I wasn't aware of the deprecation in 4.0 :) 
But still auto-registering works, since it's located in the standard namespace `BUNDLE\Command`, see https://symfony.com/doc/3.4/console.html#registering-the-command

I'm just thinking if we should leave this out of this PR for now and do the manual service definition for all Pimcore commands in a separate PR, so that we don't get into a mess (some commands defined, some not, ...)

Reply:
I don't 100% agree with you. If somebody touches a command, why not just go ahead and refactor it? You are right that all should be done, but somehow we need to start with it. and that is a start ;)